### PR TITLE
Cleanup CI command line arguments for single column edmf tests.

### PR DESCRIPTION
**Pull request by trontrytel**
## Purpose
It is hard to keep track of the different edmf configurations in the single column CI runs. This is especially true as we will be testing different implicit options, and moving cases from anelastic to compressible. A better solution to this problem would be a common config file (@dennisYatunin , @nefrathenrici). But for now, I'm just trying to just add some new line breaks into the command line args to group them better:

the first two lines being common among all test cases:
`--config column --turbconv edmf --debugging_tc true --regression_test true`
`--FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false`

the second three lines being specific to the test case we are running:
`--turbconv_case GABLS --edmf_coriolis GABLS`
`--z_elem 8 --z_stretch false --z_max 400`
`--t_end 9hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins`

the remaining lines being different for each CI job:
`--moist dry`
`--job_id compressible_edmf_gabls`
`--dt 4secs`

Closes https://github.com/CliMA/ClimaAtmos.jl/issues/1243

## To-do
- Make sure that the new line syntax is correct and all the CI cases are running.

### CODE REVIEW

**Comment by trontrytel** [comment]
https://github.com/CliMA/ClimaAtmos.jl/actions/runs/3953879687/jobs/6770615118

Looks like it's not playing well with some of the docs? @simonbyrne

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -336,104 +336,208 @@ steps:
...
-        artifact_paths: "edmf_bomex/*"
+      - label: ":balloon: SCM EDMF GABLS (Dry)"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl \
+--config column --turbconv edmf --debugging_tc true --regression_test true \
+--FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false \
+--turbconv_case GABLS --edmf_coriolis GABLS \
+--z_elem 8 --z_stretch false --z_max 400 \
+--t_end 9hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins \
+--moist dry \
+--job_id compressible_edmf_gabls --dt 4secs"
```
Comment (simonbyrne) [inline .buildkite/pipeline.yml:347]: I *think* the following should work:

```suggestion
        command: >
          julia --color=yes --project=examples examples/hybrid/driver.jl
          --config column --turbconv edmf --debugging_tc true --regression_test true
          --FLOAT_TYPE Float64 --hyperdiff false --apply_limiter false
          --turbconv_case GABLS --edmf_coriolis GABLS
          --z_elem 8 --z_stretch false --z_max 400
          --t_end 9hours --dt_save_to_sol 5mins --dt_save_to_disk 5mins
          --moist dry
          --job_id compressible_edmf_gabls --dt 4secs
```

**Review by dennisYatunin — APPROVED** [review]

**Pull request merged**